packages/transaction-controller/src/types.ts

```diff
@@ -689,3 +692,38 @@ export type DefaultGasEstimates = {
    */
   maxPriorityFeePerGas?: string;
 };
+
+/**
+ * Represents an error encountered during a transaction.
+ */
+export type TxError = {
+  /**
+   * A descriptive error name.
+   */
+  name: string;
+
+  /**
+   * A descriptive error message providing details about the encountered error.
+   */
+  message: string;
+
+  /**
+   * The stack trace associated with the error, if available.
+   */
+  stack?: string;
+
+  /**
+   * An optional error code associated with the error.
+   */
+  code?: string;
+
+  /**
+   * The rpc property holds additional information related to the error.
+   */
+  rpc?: unknown;
+
+  /**
+   * The value property holds additional information related to the error.
+   */
+  value?: unknown;
```

Comment:
Should this be removed as it's not being used in `normalizeTxError`?

Reply:
yeap, I had to accept an `Error`in `normalizedTxError` with that I  removed the `value`  property in `TransactionError`.

Reply:
Is this still pending removal?

Reply:
Apologise, I missed this one. Removed https://github.com/MetaMask/core/pull/1915/commits/c15adfbe0c05beb4a4d2a60cde0ca05853db2b2a